packages/dashboard-core-plugins/src/linker/LinkerLink.tsx

```diff
@@ -14,6 +14,7 @@ import { TableUtils } from '@deephaven/jsapi-utils';
 import memoize from 'memoize-one';
 
 import './LinkerLink.scss';
+import classNames from 'classnames';
```

Comment:
I thought we had a rule that enforced import order... but in any case, put this order at the top after the `react` import.
We import from packages first, then from local files (in this case, `LinkerLink.scss` is a local file, where `classNames` is from `node_modules` and should be at the top)

Reply:
Still need to move this import up